doc/man7/proxy-certificates.pod

```diff
@@ -0,0 +1,359 @@
+=pod
+
+=head1 NAME
+
+proxy-certificates - Proxy certificates in OpenSSL
+
+=head1 DESCRIPTION
+
+Proxy certificates are defined in
+L<RFC 3820|https://tools.ietf.org/html/rfc3820>.  They are used to
+extend rights to some other entity (a computer process, typically, or
+sometimes to the user itself).  This allows the entity to perform
+operations on behalf of the owner of the EE certificate.
+
+The requirements for a valid proxy certificate are:
+
+=over 4
+
+=item *
+
+They are issued by an End Entity, either a normal EE certificate, or
+another proxy certificate.
+
+=item *
+
+They must not have the extensions B<subjectAltName> or
+B<issuerAltName>.
```

Comment:
Move ‚extensions‘ to the end of the sentence. (That’s the correct ordering according to my limited understanding of English grammar.)

Reply:
As far as I know, this is correct English. @mattcaswell?

Reply:
@mspncp's suggestion seems the more natural to me.  Alternatively, add a colon:
> They must not have the extensions: **subjectAltName** or **issuerAltName**.

Reply:
It seems weird to me, or ambiguous at best, as "the **subjectAltName** extension" could mean that **subjectAltName** has an extension (not named here).

Regarding noun before or after a proper name (which is what we're disputing), it seems like it's hard to get an absolute agreement, but from reading the following, I'm still leaning toward what I wrote:

https://english.stackexchange.com/questions/114871/does-a-name-go-before-or-after-the-noun-it-modifies

Reply:
The suggest colon form mostly feel clumsy.  Slightly better if "the" is replaced with "these":

> They must not have these extensions: **subjectAltName** or **issuerAltName**.

Reply:
Side note: *never* had I thought that working with OpenSSL would lead me to have *English language disputes*.  Oy...

Reply:
"must not have the subjectAltName or issuerAltName extensions" is the best way to say this.

Reply:
> It seems weird to me, or ambiguous at best, as "the **subjectAltName** extension" could mean that subjectAltName has an extension (not named here).

This interpretation would usually be expressed using the genitive: "the **subjectAltName**'s extension" (equally: "the extension of the **subjectAltName**").  People, being lazy, would forget.

Reply:
Either form sounds correct to me, but @mspncp's suggestion sounds more natural and so I would prefer that form.

Reply:
...

Ok then, I give in